base_partner_company_group/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This Module adds a company group field to companies, which is also propagated to contacts and allows to search and group for the company group in contact, leads, sale orders and invoices.
```

Comment:
This description is wrong now. We should invite the user to check the other addons to get the feature enabled on leads, SO and invoices.